ihmc-pub-sub/src/main/java/us/ihmc/pubsub/attributes/ParticipantProfile.java

```diff
@@ -136,112 +136,136 @@ public ParticipantProfile discoveryServer(String discoveryServerAddress, int dis
    }
 
    /**
-    * Bind this participant to only the addresses in bindToAddressRestrictions
-    * 
-    * Functionality, this will create a new UDPv4 transport with the whitelist set to "bindToAddressRestrictions". Optionally, a shared memory transport will be added as well.
-    * useBuiltinTransports will be set to false 
-    * 
-    * @param addSharedMemoryTransport Enabled shared memory communication by adding a shared memory transport to this participant.
-    * @param bindToAddressRestrictions Limit the scope of this participant to the list of hosts. If null or empty, the participant will not be able to communicate.
-    * @return
+    * Add transport to use with this participant and register it to this participant
     */
-   public ParticipantProfile bindToAddressRestrictions(boolean addSharedMemoryTransport, List<InetAddress> bindToAddressRestrictions)
```

Comment:
I removed `bindToAddressRestrictions`. It was not used anywhere and it is essentially duplicate functionality to what is now in `addUDPv4Transport`